script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -1720,37 +1849,39 @@ It's a memento from my father. If I take those words at face value, then it's th
 [sha:2d24a2e6f67374cc549762d728af4fb9141bd555]{
 -- Page 502, Offset 34726.
 -- 「……そうですね。ありがとうございます」
-"...I guess it is. Thank you very much."
+"I guess it is... Thank you very much."
 }
 [sha:3301780d5ec8217515948d9c8ff8419f50f8a0b6]{
 -- Page 502, Offset 34727.
 -- 拾っておいてくれた先輩に感謝して、ナイフを受け取る。
-I thank Senpai for picking it up and accept the knife.
+I thank her for picking it up, and pocket it again.
 }
 [sha:1acd3c428349bbcca6e18e34594115a0ce8d56ce]{
 -- Page 503, Offset 34728.
 -- 「さて、後は地上に戻るだけですか。今の遠野くんならひとりでも瓦礫を登っていけますよね」
-"Now then, all that's left is to head back to the surface. By now, you should be capable of climbing the rubble on your own, right?"
+"Now then, all that's left is to head back to the surface.
+ You should be able to handle climbing back up on your own by now, right?"
```

Comment:
I think the 今の遠野くん is more referring to the fact that he's a goddamn vampire, so he's strong enough to get up. Remember he commented on how hard it would be to get up when he initially descended.

>The way you are now, you should be able to handle climbing back up on your own, right?